jimpass/util.py

```diff
@@ -43,21 +43,29 @@ def srun(input_cmd: str, stdin: str = None, no_output: bool = False) -> (int, st
         res = run(input_cmd, stdout=DEVNULL if no_output else PIPE, stderr=DEVNULL, encoding="utf-8", shell=True)
         return res.returncode, res.stdout
 
-
-def rofi(mode: str = "dmenu", prompt: str = None, options: list = None,
+def prun(runner: str = "rofi", mode: str = "dmenu", prompt: str = None, options: list = None,
          keybindings: list = None, args: dict = None, stdin: str = None) -> (int, str):
     """
-    Run Rofi
+    Wrapper for popup rofi/wofi
     """
-    cmd = "rofi "
+    if runner == "wofi":
+        prefix = "--"
+    else:
+        prefix = "-"
+
+    cmd = [ runner ]
+
     if mode:
-        cmd += f"-{mode} "
+        cmd.append(f"{prefix}{mode}")
     if prompt:
-        cmd += f"-p \"{prompt}\" "
+        cmd.append(f"-p \"{prompt}\"")
     if options:
-        cmd += " ".join([f"-{opt}" for opt in options]) + " "
+        cmd.append(" ".join([f"{prefix}{opt}" for opt in options]))
     if keybindings:
```

Comment:
I believe you wanted to prevent Wofi from calling keybindings as it doesn't support them. May I suggest
```suggestion
    if keybindings and runner != "wofi":
```